packages/sdk/src/v2openapi/index.ts

```
@@ -226,6 +245,8 @@ class RESTApiBuilder {
 			ChildNodes: [],
 			Directives: [],
 		});
+		const baseURL = resolveConfigurationVariable(this.dataSources[this.dataSources.length - 1].Custom.Fetch.baseUrl!);
```

We should throw an error when the user passes `new EnvironmentVariables` in combination with base URL variables.